ci/deps/azure-windows-39.yaml

```diff
@@ -36,6 +36,5 @@ dependencies:
   - xlsxwriter
   - xlwt
   - pyreadstat
-  - pip
-  - pip:
-    - pyxlsb
+  - pyxlsb
+  - zstandard
```

Comment:
we don't need this yet (open PR but will be added then)

Reply:
oops

Reply:
can you remove